app/src/main/java/ch/epfl/sweng/hostme/Apartment.java

```diff
@@ -0,0 +1,196 @@
+package ch.epfl.sweng.hostme;
+
+import java.util.Date;
+
+public class Apartment {
```

Comment:
The replacement class for this is nearly done, and will abstract some actions like pulling images and handling optional data fields.